.travis.yml

```diff
@@ -9,20 +9,19 @@
 # governing permissions and limitations under the License.
 
 language: node_js
-os:
-  - linux
-  - osx
 
 node_js:
   - "10"
-  - "10.15"
-  - "10.19"
```

Comment:
the linked ticket in the PR says "Add NodeJS versions 10.15 & 10.19 to version section" https://jira.corp.adobe.com/browse/NUI-552

Why are we removing 10.15 and 10.19?

Reply:
`10.15` and `10.19` are needed for actions.

- cli related repos:
  - os: linux, windows & mac
  - node: 10, 12, 14
- nui service code repos:
  - os: linux
  - node: 10, 12, 14 plus whatever versions we use in prod, 10.15, 10.19 right now
- used in both:
  - superset of the above